src/parsing/Parse_rule.ml

```diff
@@ -483,6 +480,10 @@ let parse_paths env key value =
     ( take_opt paths_dict env parse_string_list "include",
       take_opt paths_dict env parse_string_list "exclude" )
   in
+  if Hashtbl.length paths_dict.h > 0 then
+    yaml_error paths_dict.first_tok
```

Comment:
It's better to use error_at_key because this generates an exn that can be captured in parse_generic_ast to just skip the rule with the wrong syntax (instead of aborting the whole file).